.github/workflows/ai-review.yml

```
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: AI Code Reviewer
-        uses: pomier/ai-codereviewer@main
+        uses: pomier/ai-codereviewer@debug
```

Using a `debug` branch for the AI Code Reviewer action may introduce instability into the workflow. Ensure that the `debug` branch is stable and intended for production use. If not, consider using a tagged release or the `main` branch if it's stable.
```suggestion
uses: pomier/ai-codereviewer@main
```